Assets/Scripts/Loop/Timer.cs

```diff
@@ -0,0 +1,64 @@
+/******************************************************************
+*    Author: Elijah Vroman
+*    Contributors: Elijah Vroman
+*    Date Created: 5/29/24
+*    Description: Just a timer class with some helper methods. The time
+*    manager script does the creation, running, and deletion of these
+*******************************************************************/
+using System;
+using UnityEngine;
+
+public class Timer
+{
+    private float maxTime;
+    private float timeRemaining;
+    private bool isRunning;
```

Comment:
A fun syntax is `public bool IsRunning {get; private set;}`
this allows the variable to be publically readable but prevents external writing. It might be worth doing that instead of having a large number of getter functions at the bottom of the script

Reply:
> A fun syntax is `public bool IsRunning {get; private set;}` this allows the variable to be publically readable but prevents external writing. It might be worth doing that instead of having a large number of getter functions at the bottom of the script

Thanks! I probably should have done that. Ill leave it for now if thats ok and do that in the future